Optik/Classes/ImageViewController.swift

```diff
@@ -233,14 +238,16 @@ internal final class ImageViewController: UIViewController {
         
         if scrollView.zoomScale > scrollView.minimumZoomScale {
             // Zoom out if the image was zoomed in at all.
-            UIView.animateWithDuration(Constants.ZoomAnimationDuration,
-                                       delay: 0,
-                                       options: [],
-                                       animations: {
-                                        scrollView.zoomScale = scrollView.minimumZoomScale
-                                        self.centerImage()
+            UIView.animateWithDuration(
```

Comment:
I am trying to replicate the interaction of Photos.app which doesn't really do the zooming with spring animation.